src/modules/delete.js

```diff
@@ -0,0 +1,11 @@
+export default function deleteTodo(array) {
+  array = array.filter((obj) => obj.completed === false);
+
+  for (let i = 0; i < array.length; i += 1) {
+    array[i].index = i + 1;
+  }
+
+  localStorage.setItem('todoList', JSON.stringify(array));
+
+  return array;
+}
```

Comment:
- [ ] Hi @iKuartz, a nice job for implementing a function to clear all the tasks in the collection. Please let's add another function for deleting a task(remove an element from the array). This way, you meet the project's requirements and get closer to approval.

Reply:
I see see this has been implemented 👍🏽